rdr_service/genomic_enums.py

```diff
@@ -107,6 +107,7 @@ class GenomicJob(messages.Enum):
     METRICS_FILE_INGEST = 50
     RECONCILE_INFORMING_LOOP_RESPONSES = 51
     INGEST_RESULT_VIEWED = 52
+    RECONCILE_PDR_DATA = 53
```

Comment:
I just merged an enum with value `53`.

Reply:
And `54` too.

Reply:
Updated https://github.com/all-of-us/raw-data-repository/pull/2823/commits/6c058d65e12e047ba74bc02b0d66c88cab9737f8